wiki/Glossary/en.md

```diff
@@ -855,25 +874,25 @@ One of many reasons as to why your timing of a beatmap is important because osu!
 
```

Comment:
```
One of many reasons as to why your timing of a beatmap is important because osu! takes the given [BPM](#bpm) and offset to determine where the hit objects should go. If your timing is off, even by a little, then it is next to unrankable to map hit objects on the editor.
```